abstutil/src/io.rs

```diff
@@ -72,14 +72,18 @@ pub fn maybe_read_json<T: DeserializeOwned>(path: String, timer: &mut Timer) ->
     }
 
     timer.start(format!("parse {}", path));
-    // TODO timer.read_file isn't working here. And we need to call stop() if there's no file.
-    let result: Result<T, Error> = slurp_file(&path).and_then(|raw| {
-        serde_json::from_slice(&raw).map_err(|err| Error::new(ErrorKind::Other, err))
-    });
+    let result = maybe_read_json_without_timer(&path);
     timer.stop(format!("parse {}", path));
     result
 }
 
+pub fn maybe_read_json_without_timer<T: DeserializeOwned>(path: &String) -> Result<T, Error> {
```

Comment:
@dabreegster I didn't want to create a timer just to read a json file. Let me know if this is OK.

Reply:
You can pass `&mut Timer::throwaway()` -- it won't print to STDOUT or anything. I'd prefer that instead of increasing the already ridiculous API surface area here. (abstutil is one of those things that's grown without design for way too long)

Alternatively, nothing here is really helping you that much; maybe I misled you. If you prefer, you can just directly call `serde_json::from_slice`

Reply:
Alright, thanks for your patience. This is my first time using Rust! :)